tools/aws_lab_setup/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -22,3 +29,181 @@ ec2_lab_node_types:
 
 #  - name: haproxy
 #    type: rhel7
+
+ec2_lab_node_types_networking:
+  - name: ansible
+    type: rhel7-tower
+
+  - name: rtr1
+    type: csr-byol
+
+  - name: rtr2
+    type: csr-byol
+
+  - name: host1
+    type: rhel7
+
+# --- Lookup tables --- #
+# Region specific AMIs, subnets, and VPCs
+
+ec2_regions:
+  us-east-1:
+    vpc: vpc-06c9a463
+    subnet_id: subnet-a2169ed5
+    key_name: "{{ ec2_key_name | default('training') }}"
+    amis:
+      centos7: ami-61bbf104
+      centos7-tower: ami-877b9e91
+      rhel7: ami-85241def
+      rhel7-tower: ami-85241def
+      ubuntu14: ami-feed08e8
+      ubuntu16: ami-9dcfdb8a
+      csr-byol: ami-bcbfb9c7
+  us-east-2:
+    key_name: "{{ ec2_key_name | default('training') }}"
+    amis:
+      csr-byol: ami-b61437d3
+      rhel7: ami-cfdafaaa
+      rhel7-tower: ami-cfdafaaa
+      ubuntu14: ami-feed08e8
+      ubuntu16: ami-9dcfdb8a
+      centos7: ami-61bbf104
+      centos7-tower: ami-877b9e91
+  us-west-1:
+    vpc: vpc-ce1f9aab
+    subnet_id: subnet-25b9157c
+    key_name: "{{ ec2_key_name | default('training') }}"
+    amis:
+      centos7: ami-af4333cf
+      centos7-tower: ami-c04a00a0
+      rhel7: ami-f88fc498
+      rhel7-tower: ami-f88fc498
+      ubuntu14: ami-6ebcec0e
+      ubuntu16: ami-b05203d0
+      csr-byol: ami-99e5d0f9
+  us-west-2:
```

Comment:
I know these lookup tables need to go away, but this PR isn't do ing that. Why is data structure for reasons like us-west-2 (and others) different?

Reply:
the way the lookup is done is using a new python filter which apparently doesn't need the data structure to match to work... these were inherited by SAs who added them for their respective region.  Would you rather
- not care since it works, and we will address with the ec2_find_ami in another PR thus making all of this legacy
- change to match data structure currently

Reply:
Change to match data structure currently. I'd prefer to avoid as many unique snowflakes here and there have been long standing issues filed to support these regions on the linux (non-network-automation) side of things.

Reply:
this should match now... as soon as this PR gets accepted i will work on replacing this all with ec2_find